shared/cmd/helpers.go

```diff
@@ -70,3 +72,21 @@ func EnterPassword(confirmPassword bool, pr PasswordReader) (string, error) {
 	}
 	return passphrase, nil
 }
+
+// expandWeb3EndpointIfFile expends the path for --http-web3provider if specified as a file
```

Comment:
Capitalization and punctuation for godoc please

Reply:
ok